content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,671 @@
+
+The Prometheus _server_ collects Prometheus metrics data from targets defined in
+a configuration file. A target is an endpoint that exposes metrics for the
+Prometheus server to store.
+
+Prometheus is so ubiquitous in the monitoring space that many tools natively
+emit metrics in Prometheus format, including
+[Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/)
+and
+[HashiCorp's Nomad](https://developer.hashicorp.com/nomad/docs/operations/monitoring-nomad).
+And for those that don’t, there are a number of vendor- and community-built
+[Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/) to
+aggregate and import data into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and
+application metrics, one of its most popular use cases is to monitor Kubernetes.
+This is the aspect of Prometheus monitoring that we will focus on in this
+article.
+
+## Prometheus metrics with OpenTelemetry
+
+In this section, you'll learn about a couple of OTel Collector components that
+demonstrate the interoperability between OTel and Prometheus.
+
+First, let’s do a quick refresher on the [Collector](/docs/collector/) – it’s an
+OTel component that can be used to collect telemetry from multiple sources and
+export data to multiple destinations. The Collector also handles telemetry
+processing, such as modifying data attributes and scrubbing personally
+identifiable information. For example, you can use Prometheus SDKs to generate
+metrics, ingest them with the Collector, do some processing (if desired) and
+then forward them to your chosen backend.
+
+![Diagram showing the different components of the OTel Collector](OTel-collector-refresher.png)
+
+The
+[Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)
+allows you to collect metrics from any software that exposes Prometheus metrics.
+It serves as a drop-in replacement for Prometheus to scrape your services, and
+supports the
+[full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config)
+of configurations in `scrape_config`.
+
+If you are interested in
+[exemplars](/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded
+value that associates OTel context with a metric event, you can also use the
+Prometheus receiver to ingest them in the Prometheus format and convert it to
+OTLP format. This enables you to correlate traces with metrics.
+
+Something to consider with this component is that it is under active
+development; as such, it has several
+[limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning),
+including that it’s a stateful component. Additionally, it is recommended to not
+use this component when multiple replicas of the Collector are run, because in
+this state:
```

Comment:
```suggestion
including that it’s a stateful component. Additionally, it is not recommended to
use this component when multiple replicas of the Collector are run *without the target allocator*,
because in this state:
```